### PR TITLE
[es] Update list of versions to match English 

### PR DESCRIPTION
**Pull request by AbdelatifAitBara**
Align https://kubernetes.io/es/docs/home/supported-doc-versions/ with https://kubernetes.io/docs/home/supported-doc-versions/

[Preview of the updated page.](https://deploy-preview-46348--kubernetes-io-main-staging.netlify.app/es/docs/home/supported-doc-versions/)

/language es

### CODE REVIEW

File: data/i18n/es/es.toml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # i18n strings for the Spanish (main) site.
 
+[docs_version_latest_heading]
+other = "Versión Actual"
+
+[docs_version_other_heading]
+other = "Versiones anteriores"
```
Comment (sftim) [inline data/i18n/es/es.toml:7]: We also need `docs_version_current` (I missed this in my changes for French)

Comment (sftim) [reply data/i18n/es/es.toml:7]: Try: “(versión actual)”

Comment (AbdelatifAitBara) [reply data/i18n/es/es.toml]: Thank you, done.

**Comment by ramrodo** [comment]
Thanks!
/approve

**Pull request merged**